src/locale/vi.js

```diff
@@ -61,6 +61,8 @@ export default moment.defineLocale('vi', {
         hh: '%d giờ',
         d: 'một ngày',
         dd: '%d ngày',
+        w: 'một tuần',
+        ww: '%d tuần',
```

Comment:
Hi @1ec5, are those two lines something you can verify?

Reply:
Yes, this is correct.